recipes/pymatnest/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "pymatnest" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/libAtoms/{{ name }}/archive/master.zip
```

Comment:
If the upstream maintainers don't have an actual release I am pretty not-enthused about making up a version number here.

Reply:
What’s the alternative? Once they release an official version I definitely update it, but I doubt that is happening any time soon.